modules/navigate.js

```diff
@@ -0,0 +1,32 @@
+export const navigate = [
+  {
+    link: 'link_list',
+    target_ID: 'section_list',
+  },
+  {
+    link: 'add_link',
+    target_ID: 'add_section',
+  },
+  {
+    link: 'contact_link',
+    target_ID: 'contact_section',
+  },
+];
+
+export const navigationLinks = () => {
+  navigate.forEach((navigation) => {
+    const link = document.getElementById(navigation.link);
+
+    link.addEventListener('click', (e) => {
+      e.preventDefault();
+
+      const sections = document.querySelectorAll('main > section');
+      sections.forEach((section) => {
+        section.classList.add('_display_f_');
+      });
+
```

Comment:
- [ ] The navbar is not a requirement now, please focus on making the app work correctly.